CONTRIBUTING.md

```diff
@@ -185,7 +185,14 @@ If you are updating tests and just want to run a single test to check it, you
 can use this syntax to run it exactly as the test harness would:
 
 ```text
-$ python tools/test.py -v --mode=release parallel/test-stream2-transform
+$ python tools/test.py --mode=release parallel/test-stream2-transform
```

Comment:
>I think it would be run "exactly as the test harness would" either way given that `tools/test.py` *is* the test harness, regardless of the options.

I assume the test harness means `make test`, otherwise that part of the sentence is kinda meaningless. The reason I think this is necessary is that using `-J` can cause subtle bugs (when tests interact with each other), and that is impossible to debug if you don't use the flag by default.